packages/veritone-react-common/package.json

```diff
@@ -123,12 +125,18 @@
       "^helpers(.*)$": "<rootDir>/src/helpers$1",
       "^components(.*)$": "<rootDir>/src/components$1",
       "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/test/__mocks__/fileMock.js",
-      "\\.(css|scss)$": "identity-obj-proxy"
+      "\\.(css|scss)$": "identity-obj-proxy",
+      "^dnd-core$": "dnd-core/dist/cjs",
```

Comment:
@seta-tuha

Reply:
It's just configuration for dnd test

Reply:
I'm fine with this instance since it's just in jest